packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -279,8 +293,8 @@ export function usePress(props: PressHookProps): PressResult {
     let state = ref.current;
     let pressProps: DOMAttributes = {
       onKeyDown(e) {
-        if (isValidKeyboardEvent(e.nativeEvent, e.currentTarget) && e.currentTarget.contains(e.target as Element)) {
-          if (shouldPreventDefaultKeyboard(e.target as Element, e.key)) {
+        if (isValidKeyboardEvent(e.nativeEvent, e.currentTarget) && nodeContains(e.currentTarget, e.nativeEvent.composedPath()[0] as Element)) {
```

Comment:
Can we make a util for this, e.g. `getEventTarget`? Not sure but this may also be slower than accessing e.target since the browser will have to construct an array...